packages/core-data/src/resolvers.js

```diff
@@ -70,8 +70,8 @@ export function* getEntityRecords( kind, name, query = {} ) {
  * Requests theme supports data from the index.
  */
 export function* getThemeSupports() {
-	const index = yield apiFetch( { path: '/' } );
-	yield receiveThemeSupportsFromIndex( index );
+	const currentTheme = yield apiFetch( { path: '/wp/v2/themes/active' } );
```

Comment:
Maybe we can preload this API endpoint here https://github.com/WordPress/gutenberg/blob/master/lib/client-assets.php#L1303 ?

Reply:
Added this endpoint to that list. It correctly prevents the Featured Image editor from flashing onto the screen like it was.

Reply:
Cool, do we need to pass the `edit` context here? (and the preloaded path)?

Reply:
That I am not sure of. The default context is now `edit` because that is the only context this endpoint currently supports. But might be good to include it for when/if that default changes.

Reply:
Context was removed entirely [per the discussion in #core-restapi](https://wordpress.slack.com/archives/C02RQC26G/p1539297122000100).